requirements/base.in

```diff
@@ -9,5 +9,6 @@ Django<5.0                # Web application framework
 
 djangorestframework<4.0   # REST API
 edx-drf-extensions        # Extensions to the Django REST Framework used by Open edX
+drf_yasg                  # Generate OpenAPI 2.0 (Swagger) and 3.0.2 (OpenAPI) API documentation
```

Comment:
This is a package to create a swagger documentation for the rest api.
The edx-platform uses it via the wrapper [edX API Documentation Tools](https://github.com/openedx/api-doc-tools).
But we are creating swagger documentation here, so better remove it!

Reply:
If it's necessary to specify, it should be in `docs.in`, not `base.in`. But if we don't use it directly for creating this repo's docs, don't specify it at all.

Reply:
We don't use it here. I will remove.
Because this will be inherited in edx-platform, the docs would be show there.
I thought this was a good idea at first, but I don't think this worth the hassle.